contracts/deploy/021_solidity_upgrade.js

```diff
@@ -0,0 +1,322 @@
+        },
+        {
+          // Upgrade OUSD implementation
+          contract: cOUSDProxy,
+          signature: "upgradeTo(address)",
+          args: [dOUSD.address],
+        },
+        {
+          // Set VaultCore implementation
+          contract: cVaultProxy,
+          signature: "upgradeTo(address)",
+          args: [dVaultCore.address],
+        },
+        {
+          // Set VaultAdmin implementation
+          contract: cVault,
+          signature: "setAdminImpl(address)",
+          args: [dVaultAdmin.address],
+        },
+        {
+          // Set new trustee
+          contract: cVault,
+          signature: "setTrusteeAddress(address)",
+          args: [cBuyback.address],
+        },
+        {
+          // Collect old OGN
+          contract: cOldBuyback,
+          signature: "transferToken(address,uint256)",
+          args: [cOGN.address, ognBalance],
+        },
+        {
+          // Move old OGN forward to new contract
+          contract: cOGN,
+          signature: "transfer(address,uint256)",
+          args: [cBuyback.address, ognBalance],
+        },
+        {
+          // Set new oracle address
+          contract: cVault,
+          signature: "setPriceProvider(address)",
+          args: [dOracleRouter.address],
+        },
+        {
+          // Add CRV as a swap token
+          contract: cVault,
+          signature: "addSwapToken(address)",
+          args: [assetAddresses.CRV],
+        },
+        {
+          // Add COMP as a swap token
+          contract: cVault,
+          signature: "addSwapToken(address)",
+          args: [assetAddresses.COMP],
+        },
+        {
+          // Add AAVE as a swap token
+          contract: cVault,
+          signature: "addSwapToken(address)",
+          args: [assetAddresses.AAVE],
+        },
+        {
+          // Remove old Compound strategy
+          contract: cVault,
+          signature: "removeStrategy(address)",
+          args: [cOldCompoundStrategyProxy.address],
+        },
+        {
+          // Approve Compound strategy in Vault
+          contract: cVault,
+          signature: "approveStrategy(address)",
+          args: [cCompoundStrategyProxy.address],
+        },
+        {
+          // Add Compound as default USDT strategy
+          contract: cVault,
+          signature: "setAssetDefaultStrategy(address,address)",
+          args: [assetAddresses.USDT, cCompoundStrategyProxy.address],
+        },
+        {
+          // Addd Compound as default USDC strategy
+          contract: cVault,
+          signature: "setAssetDefaultStrategy(address,address)",
+          args: [assetAddresses.USDC, cCompoundStrategyProxy.address],
+        },
+        {
+          // Approve AAVE strategy in Vault
+          contract: cVault,
+          signature: "approveStrategy(address)",
+          args: [cAaveStrategyProxy.address],
+        },
+        {
+          // Add AAVE as default DAI strategy
```

Comment:
Let's keep this compound for now and use the strategist to prove out the AAVE strat for a bit with a smaller amount before putting lots of millions in first.

Reply:
Ok, I'll remove it as a default strategy but leave it as an approved strategy

Reply:
👍